cmd/version.go

```diff
@@ -0,0 +1,37 @@
+// Copyright © 2018 openSUSE opensuse-project@opensuse.org
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cmd
+
+import (
+	"fmt"
+
+	"github.com/spf13/cobra"
+)
+
+//Version populated from CI/CD
+var Version = "HEAD"
+
+// versionCmd represents the version command
+func newVersionCmd() *cobra.Command {
+	return &cobra.Command{
+		Use:   "version",
+		Short: "Show version of the helm mirror plugin",
+		Run:   runVersion,
+	}
+}
+
+func runVersion(*cobra.Command, []string) {
+	fmt.Println(Version)
```

Comment:
I'd also add the git commit, if available :wink:  (see how [openSUSE/umoci](https://github.com/openSUSE/umoci/blob/master/cmd/umoci/main.go) does it, for example).